v1/examples/pipeline/pipeline_created_with_pipeline_template.yaml

```diff
@@ -0,0 +1,23 @@
+# Following is an example of creating a pipeline using a pipeline template
+#  and also passing different types of inputs in the template
+
+version: 1
+kind: pipeline
+type: template
+spec:
+  id: pipeline_temp
+  versionLabel: v1
+  template_inputs:
+    input1: abc # string
+    input2: 2 # number
+    input3: true # boolean
+    input4: # array
+      - a
+      - b
+    input5: # object
```

Comment:
Add |- is not required?

Reply:
it's not required, with or without |- in both cases, yaml is valid